experiments/tf_trainer/tf_gru_attention/run.py

```diff
@@ -22,15 +22,22 @@
 tf.app.flags.DEFINE_string("embeddings_path",
                            "local_data/glove.6B/glove.6B.100d.txt",
                            "Path to the embeddings file.")
+tf.app.flags.DEFINE_boolean("is_binary_embedding", True,
```

Comment:
We should probably have the default flag match the default embedding path. So, that would make it False, right?

Reply:
Yes good point. Will change